src/main/java/seedu/ui/Ui.java

```diff
@@ -80,7 +86,35 @@ public String getCurrentUserInput() {
      */
     public void printExitMessage() {
         String reassureUser = "We'll help you remember everything you told us :)\n"
-                + "See you again!~~\n";
-        printWithSeparatorLine(reassureUser);
+                + "See you again!~~";
+        printLine(reassureUser);
+    }
+
+    /**
+     * Prints a list of ingredients, indented by four spaces.
+     * @param listOfIngredients The list of ingredients to be printed.
+     */
+    public void printListOfIngredients(ArrayList<Ingredient> listOfIngredients) {
+        int listSize = listOfIngredients.size();
+        for (Ingredient ingredient : listOfIngredients) {
+            printLine(FOUR_SPACE_INDENTATION + ingredient);
```

Comment:
Maybe can consider printing numbers before each ingredient too

Reply:
Edited the function to allow both index and no index, so it is reusable.